src/olympia/amo/middleware.py

```diff
@@ -88,6 +90,20 @@ def process_request(self, request):
         request.LANG = prefixer.locale
 
 
+class AuthenticationMiddlewareWithoutAPI(AuthenticationMiddleware):
+    """
+    Like AuthenticationMiddleware, but disabled for the API, which uses its
+    own authentication mechanism.
+    """
+    def process_request(self, request):
+        if request.path.startswith('/api/'):
```

Comment:
Don't we have an root-url endpoint from rest-framework that lives under `/api/` so that we could use `reverse` to generate this string? Just curious.